xdg-ninja.sh

```diff
@@ -1,16 +1,30 @@
 #!/usr/bin/env sh
 
+has_command() {
+    command -v "$1" >/dev/null 2>/dev/null
+    return $?
+}
+
 USE_GLOW=false
 USE_BAT=false
-if command -v glow >/dev/null 2>/dev/null; then
+USE_PYGMENTIZE=false
+USE_HIGHLIGHT=false
+if has_command glow; then
     USE_GLOW=true
-elif command -v bat >/dev/null 2>/dev/null; then
-    USE_BAT=true
-    printf "Glow not found, markdown rendering will be done by bat.\n"
-    printf "Install glow for easier reading & copy-paste.\n"
 else
-    printf "Glow or bat not found, markdown rendering not available.\n"
-    printf "Output will be raw markdown and might look weird.\n"
+    if has_command bat; then
+        USE_BAT=true
+        printf "Glow not found, markdown rendering will be done by bat.\n"
+    elif has_command pygmentize; then
+        printf "Nor glow, neither bat found, markdown rendering will be done by pygmentize.\n"
+        USE_PYGMENTIZE=true
+    elif has_command highlight; then
+        printf "Nor glow, nor bat, neither pygmentize found, markdown rendering will be done by highlight.\n"
+        USE_HIGHLIGHT=true
+    else
+        printf "Nor glow, nor bat, nor pygmentize, neither highlight found, markdown rendering not available.\n"
```

Comment:
I think this is getting too verbose. Also, I _think_ the order of nor and neither is incorrect, see [here](https://www.thefreedictionary.com/neither+...+nor), but I could be wrong.

I think this will suffice.
```suggestion
        printf "Markdown rendering will be done by bat. (Glow is recommended)\n"
    elif has_command pygmentize; then
        printf "Markdown rendering will be done by pygmentize. (Glow is recommended)\n"
        USE_PYGMENTIZE=true
    elif has_command highlight; then
        printf "Markdown rendering will be done by highlight. (Glow is recommended)\n"
        USE_HIGHLIGHT=true
    else
        printf "Markdown rendering not available. (Glow is recommended)\n"
```